include/sea_dsa/Graph.hh

```diff
@@ -99,12 +99,15 @@ public:
       const_iterator;
   typedef boost::indirect_iterator<typename NodeVector::iterator> iterator;
   typedef ValueMap::const_iterator scalar_const_iterator;
+
   typedef boost::filter_iterator<IsGlobal, typename ValueMap::const_iterator>
       global_const_iterator;
   typedef ArgumentMap::const_iterator formal_const_iterator;
   typedef ReturnMap::const_iterator return_const_iterator;
   using alloc_site_iterator
     = boost::indirect_iterator<typename AllocSites::iterator>;
+  using dsa_call_site_iterator
+    = boost::indirect_iterator<typename DsaCallSites::iterator>;
```

Comment:
const iterators are good to have as well. You've placed your code between one for `AllocSite`, better move it down